### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by kelidon**

### CODE REVIEW

File: app/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter.
+flutter:
+
+  # The following line ensures that the Material Icons font is
+  # included with your application, so that you can use the icons in
+  # the material Icons class.
+  uses-material-design: true
+  assets:
```
Comment (captainhorsepower) [inline app/pubspec.yaml]: ты забыл добавить в гит

Error: unable to find directory entry in pubspec.yaml: /Users/artemvrby/projects/bsu/pipercrux/app/graphics/

File: app/lib/widgets/content/view/file-tile.component.dart [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:pipercrux/widgets/content/model/content.model.dart';
+import 'package:provider/provider.dart';
+import 'package:line_awesome_icons/line_awesome_icons.dart';
+
+class FileTileComponent extends StatelessWidget {
```
Comment (captainhorsepower) [inline app/lib/widgets/content/view/file-tile.component.dart:6]: Component suffix is kinda redundant, but may be not.

File: app/lib/widgets/content/view/content.view.dart [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    final homeModel = Provider.of<ContentModel>(context);
+
+    List<Widget> _widgetOptions = <Widget>[
+      FilesListView(),
+      //TestList(),
```
Comment (captainhorsepower) [inline app/lib/widgets/content/view/content.view.dart]: remove commented out stuff pls)0

File: app/lib/widgets/authorization/view/auth.view.dart [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:pipercrux/widgets/authorization/model/auth.model.dart';
```
Comment (captainhorsepower) [inline app/lib/widgets/authorization/view/auth.view.dart]: it's actually authentication, which means acknowledging the user
not authorization, which means verifying user's permission to do smth

i mean just rename folder

**Review by captainhorsepower — CHANGES_REQUESTED** [review]

**Review by captainhorsepower — APPROVED** [review]

File: app/lib/widgets/content/view/file-tile.component.dart (hunk 2 above)
Comment (kelidon) [reply app/lib/widgets/content/view/file-tile.component.dart:6]: that means it's a small widget which is just a part of some widget with 'view' suffix

**Pull request merged**